packages/astro/src/assets/vite-plugin-assets.ts

```diff
@@ -197,7 +197,7 @@ export default function assets({
 				resolvedConfig = viteConfig;
 			},
 			async load(id) {
-				if (/\.(heic|heif|avif|jpeg|jpg|png|tiff|webp|gif)$/.test(id)) {
+				if (/\.(jpeg|jpg|png|tiff|webp|gif|svg)$/.test(id)) {
```

Comment:
For the removed formats here: It turns out that `image-size` does not actually support them! The popular alternative `probe-image-size` does, but it's also multiple times larger... 

It's unlikely that people will import heic and heif, but avif could happen. Maybe we could send a fix upstream? Or maybe we could shake the world with a new `image-size` replacement, who knows.